ui/components/app/currency-input/currency-input.js

```diff
@@ -25,12 +25,14 @@ import { getCurrentCurrency, getShouldShowFiat } from '../../../selectors';
  * @param options0.featureSecondary
  * @param options0.onChange
  * @param options0.onPreferenceToggle
+ * @param options0.primaryNumberOfDecimals
  */
 export default function CurrencyInput({
   hexValue,
   featureSecondary,
   onChange,
   onPreferenceToggle,
+  primaryNumberOfDecimals,
```

Comment:
Should we throw an error if `primaryNumberOfDecimals` is more than 16?

Reply:
@mcmire `primaryNumberOfDecimals` value sent from <SendAmountRow/> will either be 15 ( as we have set at the moment) or any value less than that, and we can also default to 8 (as it was the value we are currently replacing) or to 15 (to avoid confusion).

Reply:
I guess I want to make sure anyone who uses this in the future isn't confused. Is there anything stopping them from saying `<CurrencyInput primaryNumberOfDecimals={100} />`?